ext/drwrap/drwrap.c

```diff
@@ -974,6 +974,13 @@ drwrap_exit(void)
     if (count != 0)
```

Comment:
- In the title: s/i3/i#3/.
- In the title: if it's a single-commit issue, there is no mini-summary before the colon.
- In the description: s/Fixes i/Fixes #/; s/1.//.
- Please augment the description with what kind of resources (unregistering for events) and why it matters.

Reply:
Got it!